api/proto/certs/certs.proto

```diff
@@ -6,17 +6,6 @@ syntax = "proto3";
 option go_package  = "github.com/lf-edge/eve/api/go/certs";
 option java_package = "com.zededa.cloud.uservice.proto";
 
-// This is the request payload for POST /api/v1/edgeDevice/EdgeNodeCerts
-// or /api/v2/edgeDevice/EdgeNodeCerts
-// ZEdgeNodeCert carries a set of X.509 certificate and their properties
-// from EVE to Controller.
-// The message is assumed to be protected by a TLS session bound to the
-// device certificate for v1.
-// The message is assumed to be protected by signing envelope for v2.
-message ZEdgeNodeCert {
-  repeated ZCert certs = 1;   //list of certificates sent by edge node
-}
-
 // This is the response payload for GET /api/v1/edgeDevice/ControllerCerts
 // or /api/v2/edgeDevice/ControllerCerts
```

Comment:
Did the API endpoint change from ControllerCerts to certs as part of this change? I thought I saw that in the cloud code.

Reply:
I changed ControllerCerts to certs, since there is no EdgeNodeCerts to distinguish now.

Reply:
But this comment is then incorrect, right?

Reply:
updated comment.